charts/posthog/templates/_snippet-initContainers-wait-for-service-dependencies.tpl

```diff
@@ -5,7 +5,8 @@
   command:
     - /bin/sh
     - -c
-    - >
+    - |
+        set -e
```

Comment:
Ditto

Reply:
not sure what's the best way, but if there's any other problems here they will just be bypassed (see description)

Reply:
This is an init container to verify service dependency (e.g. is ClickHouse up and running?)

I don't get why we are adding a `set -e` here but maybe I've misunderstood something

Reply:
If you look at the description "I'm not sure how this was expected to work before, but adding set -e ..." until "Type of change" does that explain it? I can remove it from here, but if we make a typo in some line it would just be ignored and look as all is well.

Reply:
If we `set -e` and ClickHouse is not ready the script instead of looping will `exit 1`

Reply:
why would it be `exit 1` in that case? It's the same outcome if I have 
```
set -e
until (false);
do 
  echo "with set -e"
done
```

as 
```
until (false);
do 
  echo "without set -e"
done
```
As in I saw the echo output from the init container

Reply:
My bad, I thought it was propagated to the subshell

Reply:
I'll let you decide if you want to have it anywhere else, shouldn't be part of this PR anyway